static/css/_data-table.scss

```diff
@@ -6,6 +6,113 @@ $baseline-grid: 1px;
     margin: 8px;
 }
 
+// Boomerang
+.breadcrumb,
+.breadcrumb a {
+  height: 33px;
+  display: inline-block;
+}
+
+.breadcrumb { 
+  margin-left: 10px;
+  margin-bottom: 2px;
+  border-width: 2px;
+  border-style: solid;
+  overflow: hidden;
+}
+
+.breadcrumb a {
+  text-rendering: optimizeLegibility;
+  line-height: 33px;
+  font-size: 12px;
+  text-decoration: none;
+  padding: 0 10px 0 20px;
+  position: relative;
+}
+
+.breadcrumb a:after,
+.breadcrumb a:before { 
+  content: ''; 
+  display: block; 
+  width: 0; 
+  height: 0;
+  border-top: 17px solid transparent;
+  border-bottom: 17px solid transparent;
+  border-left: 10px solid transparent;
+  position: absolute;
+  margin: auto;
+  top: 0;
+  bottom: 0;
+  left: 100%;
+  z-index: 2;
+}
+
+.breadcrumb a:before { 
+  margin-left: 1px;
+  z-index: 1;
+}
+
+.breadcrumb a:last-child {
+  color: #fff;
+}
+
+.breadcrumb a:last-child:after,
+.breadcrumb a:last-child:before,
+.breadcrumb input[type=text] {
+   border: none;
+}
+.breadcrumb input[type=text] {
+  -webkit-appearance: none;
+  display: inline-block;
+  text-indent: 24px;
+  padding: 0 16px;
+  height: 33px;
+  width: 10px;
+  outline: 1px solid transparent;
+  background-origin: content-box;
+  background-position: left 50%;
+  background-repeat: no-repeat;
+  background-size: 15px 15px;
+  transition-property: all;
+  transition-duration: .13s;
+  transition-timing-function: cubic-bezier(0.455, 0.030, 0.515, 0.955);
+}
+
+/* LIGHT COLORS */
+.light .breadcrumb {
+  border-color: #304269;
+}
+
+.light .breadcrumb a {
+  color: #304269;
+  background-color: #FFF;
+} 
+.light .breadcrumb a:after {
+  border-left-color: #FFF;
+}
+.light .breadcrumb a:before { 
+  border-left-color: #9C9E9E;
```

Comment:
you can do rgba($name_of_the_color, 0.3) for the disabled ones or just play with the alpha a bit